### DISCUSSION TITLE
large error when setting StressForceOutput for training

### DISCUSSION THREAD

**Discussion by phys-chem** [mir-group/nequip#385] [Q&A] [answered]
I want to get stress for NPT simulations. I change ForceOutput to StressForceOutput, but the validation error becomes extremely large. My yaml file is attached. How to solve it? Thank you very much! 
[example.txt](https://github.com/mir-group/nequip/files/13403262/example.txt)

**Comment by Linux-cpp-lisp** [comment, marked as answer]
Neither ForceOutput nor StressForceOutput should go after RescaleEnergyEtc;  see `example.yaml` in Allegro for an example.

**Reply by phys-chem** [reply]
Thank you very much!
